README.md

```diff
@@ -1,9 +1,106 @@
-This is a [demo app](https://web.dev/porting-libusb-to-webusb/) running gPhoto2 on the Web:
+# Web-gPhoto2
 
-![A picture of DSLR camera connected via a USB cable to a laptop. The laptop is running the Web demo mentioned in the article, which mirrors a live video feed from the camera as well as allows to tweak its settings via form controls.](https://web-dev.imgix.net/image/9oK23mr86lhFOwKaoYZ4EySNFp02/MR4YGRvl0Z9AWT6vv3sQ.jpg?auto=format&w=1600)
+A gPhoto2 implementation using WebAssembly to control DSLR cameras from the browser.
 
 Powered by a [custom fork](https://github.com/RReverser/libgphoto2) of [libgphoto2](https://github.com/gphoto/libgphoto2), the [WebUSB](https://github.com/WICG/webusb) backend of [libusb](https://github.com/libusb/libusb), and WebAssembly via [Emscripten](https://emscripten.org/).
 
+# NPM
+
+## Installation
+
+```bash
+npm install web-gphoto2
+// or
+yarn add web-gphoto2
+```
+
+## Usage
+
+A short example on how to use this package:
+
+```ts
+import Camera from "web-gphoto2";
+
+let camera = new Camera();
+
+async function connectCamera() {
+  await camera.showCameraPicker();
+  await camera.connect();
+}
+
+async function getSupportedOps() {
+  const ops = await camera.getSupportedOps();
+  console.log("Supported Ops:", ops);
+}
+
+async function getCameraConfig() {
+  const config = await camera.getConfig();
+  console.log("Config:", config);
+}
+
+async function updateConfig() {
+  await camera.setConfigValue("iso", "800");
+}
+
+async function capturePreviewAsBlob() {
+  const blob = await camera.capturePreviewAsBlob();
+  console.log("Blob:", blob);
```

Comment:
Let's show a bit more realistic example here. Logged blob or file is not very useful, we could instead show e.g. creating `img` element from it.

Reply:
Added